### PR TITLE
Collaborator fields added

### PR DESCRIPTION
**Pull request by andrewconner**
Should be safe and backwards compatible everywhere. Tests updated.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/ext/ExtLibraryController.scala [hunk 1]
```diff
@@ -44,14 +44,17 @@ class ExtLibraryController @Inject() (
...
   def getLibraries() = UserAction { request =>
-    val datas = libraryCommander.getLibrariesUserCanKeepTo(request.userId) map { lib =>
-      val owner = db.readOnlyMaster { implicit s => basicUserRepo.load(lib.ownerId) }
-      LibraryData(
-        id = Library.publicId(lib.id.get),
-        name = lib.name,
-        color = lib.color,
-        visibility = lib.visibility,
-        path = Library.formatLibraryPath(owner.username, lib.slug))
+    val datas = libraryCommander.getLibrariesUserCanKeepTo(request.userId) map {
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/ext/ExtLibraryController.scala:47]: what happens if a user is keeping a page that is already in the library? (for user's own keeps, keeper already shows kept, so no problem like this)

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/ext/ExtLibraryController.scala:47]: It'll show as kept, as well. Here's what I see in Stephen's library, I didn't keep it (he did).

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/ext/ExtLibraryController.scala:47]: The larger point here is that collaborators are treated equal to owners in most cases. They can unkeep, keep, etc at will. Think of it like a Dropbox shared folder.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/ext/ExtLibraryController.scala:47]: Great!

**Pull request merged**